projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html

```diff
@@ -33,13 +33,15 @@
     <ng-container *ngIf="isHoldingMatchUserLibraryPID">
       <button *ngIf="permissions.canRequest && permissions.canRequest.can; else notRequest"
               type="button" class="btn btn-sm btn-outline-primary " (click)="addRequest(item.metadata.pid)"
-              title="{{ 'Item request' | translate}}">
+              title="{{ 'Item request' | translate}}"
+              [attr.id]="item.metadata.barcode | idAttribute:{prefix: 'item', suffix: 'request'}">
```

Comment:
Same idea than previous comment. Don't focus on only what's needed right now, think of other elements of the row or component. However the id won't be displayed on an `ng-container`. It must be on a `div` or other html tag.